brion/morphology.cpp

```diff
@@ -69,27 +69,27 @@ CellFamily Morphology::getCellFamily() const
     return _impl->plugin->getCellFamily();
 }
 
-Vector4fsPtr Morphology::readPoints() const
+Vector4fsPtr Morphology::readPoints()
```

Comment:
Then the version number should be bumped for consistency. Even if we are not strict about it at times you should try to be more disciplined about it.